spec/src/main/doc/core/implementation.asciidoc

```diff
@@ -730,7 +494,7 @@ A bean constructor may have any number of parameters. All parameters of a bean c
 
 === Injected fields
 
-An _injected field_ is a non-static, non-final field of a bean class or of any Java EE component class supporting injection.
+An _injected field_ is a non-static, non-final field of a bean class.
```

Comment:
So I removed all "* supporting injection" in core except in injectionandresolution.asciidoc (line 347 and 357) and in spi.asciidoc (line 785).
I let them because you could imagine having a non contextual object instantiated by the BM thru `CreateInjectionTarget` or `Unmanaged` and thus supporting injection.
Let me know what you think about this @jharting.